tests/publiccloud/run_ltp.pm

```diff
@@ -132,6 +145,9 @@ sub run {
 
     my $runltp_ng_repo = get_var("LTP_RUN_NG_REPO", "https://github.com/metan-ucw/runltp-ng.git");
     my $runltp_ng_branch = get_var("LTP_RUN_NG_BRANCH", "master");
+    record_info('LTP CLONE REPO', "Repo: " . $runltp_ng_repo . "\nBranch: " . $runltp_ng_branch);
+    die('Not valid python LTP_RUN_NG_REPO provided')
+      if ($ltp_runtime =~ /^python/i and $runltp_ng_repo =~ /\/metan-ucw\//i);
```

Comment:
I added this check to prevent omission of the REPO value in case of python test, where record_info reports the values helpful for debugging of crash case too.

Reply:
Please add a comment stating that this is temporary and this will be removed when the transition is completed.